spartan/aztec-network/values.yaml

```diff
@@ -15,6 +15,9 @@ network:
   # This is only applicable to GKE deployments
   gke: false
 
+  # If true then configured bootnodes are considered non-gossiping peers
+  usingGossipBootnodes: true
```

Comment:
OCD nit, but I'd name this same as the env var. I know we're not doing it with most variables, but I think it'd make it easier to find later, especially in untyped yaml files.

Reply:
Done